index.bs

```diff
@@ -2494,7 +2494,8 @@ Council Deliberations</h5>
 	<em class=rfc2119>may</em> write a <dfn>Minority Opinion</dfn>
 	explaining the reason for their disagreement.
 
-	The deliberations of the [=W3C Council=] are confidential to the [=W3C Council=].
+	The deliberations of the [=W3C Council=] are confidential to the [=W3C Council=]
+	and its [=Council Team Contact=].
 
 	If the [=W3C Council=] is unable to come to a conclusion within 45 days of being <a href="#council-convention">convened</a>,
 	the [=W3C Council Chair=] <em class=rfc2119>must</em> inform the [=AC=] of this delay
```

Comment:
There are probably other occurrences of the definite article, `the`, which should be replaced by the indefinite article, `a`, given that there may be multiple `W3C Councils` operating at any given moment. I can review the doc for these, if desired, or leave it to you, @frivoal.
```suggestion
	The deliberations of a [=W3C Council=] are confidential to the [=W3C Council=]
	and its [=Council Team Contact=].

	If a [=W3C Council=] is unable to come to a conclusion within 45 days of being <a href="#council-convention">convened</a>,
	the [=W3C Council Chair=] <em class=rfc2119>must</em> inform the [=AC=] of this delay
```

Reply:
Agree - maybe a separate issue for checking and fixing that all instances of "W3C Council" are clear that there is not a single W3C Council that meets repeatedly for different topics, but that each is considered distinct.

Reply:
@TallTed I'd suggest looking at this separately. While this PR includes one case of a vs the, this stylistic difference is broader. Could you file a separate issue?

Reply:
See #710